go.mod

```diff
@@ -5,6 +5,7 @@ go 1.18
 require (
 	github.com/Azure/azure-sdk-for-go v56.3.0+incompatible
 	github.com/Shopify/logrus-bugsnag v0.0.0-20171204204709-577dee27f20d
+	github.com/andybalholm/brotli v1.0.4
```

Comment:
TBH, I never heard of `brotli` - is it common? (looks like it's quite new).

As to the choice of dependencies; I see Klauspost mentions https://github.com/dsnet/compress as a good quality module for this type of compression (see https://github.com/klauspost/compress#other-packages).

As mentioned; I'm not very familiar with the compression, and not sure if there's already a "defacto" (or preferred) implementation for it. I _do_ consider Klauspost a good source of information (as being quite notable on compression implementations).

- Was there a specific reason to pick https://github.com/andybalholm/brotli ? 
- Do we need more eyes to review both choices, to pick one?
- Should adding brotli be done in a separate PR (giving some time to decide?)

Reply:
dsnet/compress says "This library is in active development" and only support uncompressing a `br` stream - could be fine for this PR, but we will need another dependency to implement compression by registry